rasa/cli/test.py

```diff
@@ -115,29 +116,54 @@ def run_core_test(args: argparse.Namespace) -> None:
     )
 
 
-def run_nlu_test(args: argparse.Namespace) -> None:
-    """Run NLU tests."""
+async def run_nlu_test_async(
+    config: Optional[Union[Text, List[Text]]],
+    data_path: Text,
+    models_path: Text,
+    output_dir: Text,
```

Comment:
this can also be `None`, no?
```suggestion
    output_dir: Optional[Text],
```

Reply:
Same as above

Reply:
👍🏻